src/main/java/org/pac4j/undertow/DemoHandlers.java

```diff
@@ -138,8 +146,24 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
                 sb.append("<input type=\"submit\" name=\"submit\" value=\"Submit\" />");
                 sb.append("</form>");
 
-                exchange.getResponseSender().send(sb.toString());
+                returnHtml(exchange, sb.toString());
             }
         };
     }
+
```

Comment:
In fact, it doesn't even work: I just kept the source code to remember.

When trying to access a protected area, if you give up during the authentication process (like for Twitter), a 403 page is returned: it's a blank page here and I'd like to set my own 403 page...